app/navigation/Navigator.scala

```diff
@@ -168,9 +168,9 @@ class Navigator @Inject() () extends Logging {
           userAnswers,
           HaveTradingNamePage,
           controllers.organisation.routes.BusinessTradingNameWithoutIDController.onPageLoad(CheckMode),
-          controllers.organisation.routes.NonUKBusinessAddressWithoutIDController.onPageLoad(CheckMode)
+          controllers.routes.CheckYourAnswersController.onPageLoad()
         )
-    case BusinessTradingNameWithoutIDPage  => _ => controllers.organisation.routes.NonUKBusinessAddressWithoutIDController.onPageLoad(CheckMode)
+    case BusinessTradingNameWithoutIDPage  => userAnswers => controllers.organisation.routes.NonUKBusinessAddressWithoutIDController.onPageLoad(CheckMode)
```

Comment:
`userAnswers` isn't used here and could still kept as wildcard `_`

Reply:
Removed this